.github/workflows/notebook-test.yml

```diff
@@ -76,46 +56,33 @@ jobs:
           install-linux-deps: ${{ steps.linux-changed-files.outputs.any_changed == 'true' || steps.changed-config.outputs.any_changed == 'true' }}
           ibm-quantum-token: ${{ secrets.IBM_QUANTUM_TEST_TOKEN }}
 
-      - name: Check lint
-        shell: python
-        run: |
-          import subprocess, sys
-          files = """${{ steps.all-changed-files.outputs.all_changed_files }}"""
-          args = ["tox", "-e", "lint", "--"] + files.split("\n")
-          try:
-            subprocess.run(args, check=True)
-          except:
-            print(
-              "To fix, install `tox` and run `tox -e fix` or download the fixed "
-              "notebook from this run by clicking \"Summary\" on the upper-left "
-              "of this page, and downloading the \"Executed notebooks\" artifact."
-              "\n\n"
-              "For more information, see https://github.com/Qiskit/documentation/blob/main/README.md#lint-notebooks"
-            )
-            sys.exit(1)
-
       - name: Execute notebooks
-        if: "!cancelled()"
+        if: ${{ success() && !cancelled() }}
```

Comment:
I think this is now just the default behaviour if an `if` condition isn't included. For context, we added `!cancelled()` for https://github.com/Qiskit/documentation/pull/897#pullrequestreview-1904336782

```suggestion
```